docs/tutorials/quick-start-local.rst

```diff
@@ -39,28 +39,28 @@ and details of the server. For printing cluster's status outside the cluser, its
 
     /sashab/rh-basic-cpu
     😈 Runhouse Daemon is running 🏃
-    Runhouse v0.0.28
-    server pid: 29395
+    Runhouse v0.0.34
+    🤖 aws m6i.large cluster | 🌍 us-east-1 | 💸 $0.096/hr
+    server pid: 29477
     • server port: 32300
-    • den auth: True
+    • den auth: False
     • server connection type: ssh
     • backend config:
-      • resource subtype: OnDemandCluster
-      • domain: None
-      • server host: 0.0.0.0
-      • ips: ['52.207.212.159']
-      • resource subtype: OnDemandCluster
-      • autostop mins: autostop disabled
+        • resource subtype: OnDemandCluster
+        • domain: None
+        • server host: 0.0.0.0
+        • ips: ['52.91.194.125']
+        • autostop mins: autostop disabled
+    CPU Utilization: 5.4%
     Serving 🍦 :
     • _cluster_default_env (runhouse.Env)
-      This environment has only python packages installed, if such provided. No resources were found.
-    • sd_env (runhouse.Env) | pid: 29716 | node: head (52.207.212.159)
-      CPU: 0.0% | Memory: 0.13 / 8 Gb (1.65%)
-      This environment has only python packages installed, if such provided. No resources were found.
-    • np_pd_env (runhouse.Env) | pid: 29578 | node: head (52.207.212.159)
-      CPU: 0.0% | Memory: 0.13 / 8 Gb (1.71%)
-      • /sashab/summer (runhouse.Function)
-      • mult (runhouse.Function)
+        This environment has only python packages installed, if such provided. No resources were found.
```

Comment:
can we add to this PR changing the text from "if such provided" to "if provided"?